homeworks/max.snischuk_snischuk/PokemonProject/pokemons.html

```diff
@@ -44,7 +44,8 @@
                     <fieldset class="filters__group">
                         <input class="filters__input" id="all" type="radio" name="radio-options" value="all">
                         <label class="filters__custom filters__custom--radio" for="all">All</label>
-                        <input class="filters__input" id="favorites" type="radio" name="radio-options" value="favorites">
+                        <input class="filters__input" id="favorites" type="radio" name="radio-options"
```

Comment:
Your logic of filtering broken a bit:
1. Check Small checkbox
2. Type s in the search
3. I see big pokemons which go to hidden when the input lost focus (any click)

![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/d3362500-fafc-4bb7-98c4-a0326eeed1d6)

Reply:
It is good, that you used `featured` prop of the Pokemons for the filtering by radio, but not a user can't understand, what is going on, when they hit `Favorite` radio.
Would be good to add some visual representation to the card (e.g. heart icon or another badge)
![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/c72e1474-d908-4050-8bb6-90077f22e149)